src/components/change-requests/change-request-details/change-request-details/change-request-details.test.tsx

```diff
@@ -91,81 +80,81 @@ describe('Change request details common display element tests', () => {
-      expect(screen.getByText(`${explanation.reason}`)).toBeInTheDocument();
-      expect(screen.getByText(`${explanation.explain}`)).toBeInTheDocument();
-    });
-  });
-
-  it('Renders impact section', () => {
-    const cr: StandardChangeRequest = exampleStandardChangeRequest;
-    renderComponent(cr);
-    expect(screen.getByText(`Impact`)).toBeInTheDocument();
-    expect(screen.getByText(`Scope Impact`)).toBeInTheDocument();
-    expect(screen.getByText(`${cr.scopeImpact}`)).toBeInTheDocument();
-    expect(screen.getByText(`Timeline Impact`)).toBeInTheDocument();
-    expect(screen.getByText(`${cr.timelineImpact} weeks`)).toBeInTheDocument();
-    expect(screen.getByText(`Budget Impact`)).toBeInTheDocument();
-    expect(screen.getByText(`$${cr.budgetImpact}`)).toBeInTheDocument();
-  });
-});
-
-describe('Change request details activation cr display element tests', () => {
-  it('Renders project lead', () => {
-    const cr: ActivationChangeRequest = exampleActivationChangeRequest;
-    renderComponent(cr);
-    expect(screen.getByText(`Project Lead`)).toBeInTheDocument();
-    expect(
-      screen.getByText(`${cr.projectLead.firstName} ${cr.projectLead.lastName}`)
-    ).toBeInTheDocument();
-  });
-
-  it('Renders project manager', () => {
-    const cr: ActivationChangeRequest = exampleActivationChangeRequest;
-    renderComponent(cr);
-    expect(screen.getByText(`Project Manager`)).toBeInTheDocument();
-    expect(
-      screen.getByText(`${cr.projectManager.firstName} ${cr.projectManager.lastName}`)
-    ).toBeInTheDocument();
-  });
-
-  it('Renders start date', () => {
-    const cr: ActivationChangeRequest = exampleActivationChangeRequest;
-    renderComponent(cr);
-    expect(screen.getByText(`Start Date`)).toBeInTheDocument();
-    expect(screen.getByText(`${cr.dateSubmitted.toUTCString()}`)).toBeInTheDocument();
-  });
-
-  it('Renders confirm details', () => {
-    const cr: ActivationChangeRequest = exampleActivationChangeRequest;
-    renderComponent(cr);
-    expect(screen.getByText(`Confirm WP Details`)).toBeInTheDocument();
-    expect(screen.getByText(`${cr.confirmDetails ? 'YES' : 'NO'}`)).toBeInTheDocument();
-  });
-});
-
-describe('Change request details stage gate cr display element tests', () => {
-  it('Renders confirm completed', () => {
-    const cr: StageGateChangeRequest = exampleStageGateChangeRequest;
-    renderComponent(cr);
-    expect(screen.getByText(`Confirm WP Completed`)).toBeInTheDocument();
-    expect(screen.getByText(`${cr.confirmDone ? 'YES' : 'NO'}`)).toBeInTheDocument();
-  });
-
-  it('Renders leftover budget', () => {
-    const cr: StageGateChangeRequest = exampleStageGateChangeRequest;
-    renderComponent(cr);
-    expect(screen.getByText(`Leftover Budget`)).toBeInTheDocument();
-    expect(screen.getByText(`$${cr.leftoverBudget}`)).toBeInTheDocument();
-  });
-});
+// describe('Change request details standard cr display element tests', () => {
```

Comment:
I'd say for all these tests you don't need to remove them all from the parent component, just maybe slim them down a bit. Have at least 1 sanity check test for each type of CR just to ensure the code gets run via a test?

Reply:
Sure, sounds good.